ffi/blitbuffer.lua

```diff
@@ -139,7 +139,10 @@ void *malloc(int size);
 void free(void *ptr);
 ]]
 
-local use_cblitbuffer, cblitbuffer = pcall(ffi.load, 'blitbuffer')
+-- NOTE: Try the C blitter, unless it was disabled by the user
+if G_reader_settings:nilOrFalse("dev_no_c_blitter") then
+    local use_cblitbuffer, cblitbuffer = pcall(ffi.load, 'blitbuffer')
+end
```

Comment:
I really don't mind it. But there seems to be no access to G_reader_settings in base yet.
Not sure if that's a rule, or just luck :) @Frenzie ?

Reply:
That is indeed intentional, also see https://github.com/koreader/koreader-base/commit/7fb42187478cc4c05756ab744a02b3369f140f4a

Reply:
So, what to do? Not going to lie, not a massive fan of the seven layers of indirection for something like that... ;).

Reply:
I think the real problem is, if our base/ffi/ modules are to be used for other things, `G_reader_settings` possibly not being set, and accessing `G_reader_settings:nilOrFalse` cause a "can't index a nil value" crash.
So, set an env variable from frontend (dunno if that's possible), or koreader.sh (i know it's ugly, but you grep for dev_startup_no_fbdepth), and use `os.getenv()` (commonly used in base) ? Or just set a plain boolean global variable with no risk of crash?

Reply:
Which, incidentally, is exactly what happens in the testsuite ;).

And, no, there's no setenv in Lua because it's POSIX, not ANSI C89 :/.

Reply:
Ohkay, after cursing at Linux's putenv that doesn't take a const, switched to setenv over FFI, as some stuff was already using it on Android & Windows ;).

Reply:
Yeah, functionally speaking if g and g:bla() would be fine. Architecturally speaking you probably wouldn't want that to be the only way to play with it.

Reply:
env var it is, then ;). Thanks!